src/coreclr/vm/jitinterface.cpp

```diff
@@ -13685,7 +13671,13 @@ BOOL LoadDynamicInfoEntry(Module *currentModule,
             // aren't quite set up to accept that. Furthermore the call sequences would be different - at
             // the moment an indirection cell uses "call [cell-addr]" on x86, and instead we would want the
             // euqivalent of "call [[call-addr]]".  This could perhaps be implemented as "call [eax]" </REVISIT_TODO>
-            result = pMgr->GetCallStub(ownerType, slot);
+            DispatchToken token = VirtualCallStubManager::GetTokenFromFromOwnerAndSlot(ownerType, slot);
+
+            INTERFACE_DISPATCH_CACHED_OR_VSD(
+                return FALSE; // R2R interface dispatch currently only supports fixups with a single pointer, return FALSE to skip using the method
```

Comment:
The PR has changes to add this support. Did you hit any roadblocks with making this actually work?

Reply:
I did not. But the support is only for fixups which are used directly for dispatch. This code path is used when gathering a fixup as a function pointer to use directly such as embedded into a delegate or something. I believe that this logic may only actually used for R2R versioning where we replace a non-virtual method with a virtual method and construct a delegate. I'm not entirely sure. At the very least I should write a clearer comment. I'll also see if I can actually trigger this scenario. It's quite possible that the existing R2R behavior with VSD is actually not compliant with our latest VM logic.

Reply:
This is actually completely dead code. The VIRTUAL_ENTRY_SLOT path was specific to NGEN. (So are the VIRTUAL_ENTRY_DEF/REF_TOKEN paths, but those might actually be useful for R2R as a load simplification, so I'm not touching those. I'm going to delete the logic that implements VIRTUAL_ENTRY_SLOT.